config/config.go

```diff
@@ -0,0 +1,45 @@
+package config
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+
+	"github.com/davecgh/go-spew/spew"
+
+	"github.com/spf13/viper"
+)
+
+type config struct {
+	CSV struct {
+		Path string
+	}
+	Server struct {
+		Port string
+	}
+}
+
+var C config
+
+func ReadConfig() {
```

Comment:
I'd suggest changing the function signature to return a config pointer and an error; in the place where you are using it, handle the error (if that's the case), or simply use the obtained config.
```suggestion
func ReadConfig() (*config, error) {
```